traveler-ios-ui/TravelerKitUI/ViewControllers/OrderDetailViewController.swift

```diff
@@ -0,0 +1,94 @@
+//
+//  OrderDetailViewController.swift
+//  TravelerKitUI
+//
+//  Created by Omar Padierna on 2019-05-23.
+//  Copyright © 2019 GuestLogix. All rights reserved.
+//
+
+import UIKit
+import TravelerKit
+
+let orderDetailCellIdentifier = "orderDetailCellIdentifier"
+let productCellIdentifier = "productCellIdentifier"
+let billingCellIdentifier = "billingCellIdentifier"
+let cancelOrderCellIdentifier = "cancelOrderCellIdentifier"
+
+class OrderDetailViewController: UITableViewController {
+    public var order: Order?
+
+    private var numberOfRows = 0
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        tableView.tableFooterView = UIView()
+        numberOfRows = (order?.products.count ?? 0) + 3
+    }
+
+    // MARK: - Table view data source
+
+    override func numberOfSections(in tableView: UITableView) -> Int {
+        return 1
+    }
+
+    override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return numberOfRows
+    }
+
+    override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
```

Comment:
Wouldnt you think it would be easier to use tableview headers and footers and keep the rows just for the products? This logic is very indexPath based and its not super clear what goes where. Plus it would eliminate the need for extra cells